alert-common/src/main/java/com/synopsys/integration/alert/common/rest/api/ConfigurationCrudHelper.java

```diff
@@ -27,95 +27,142 @@
 import com.synopsys.integration.function.ThrowingSupplier;
 
 public class ConfigurationCrudHelper {
-
-    private final Logger logger = LoggerFactory.getLogger(ConfigurationCrudHelper.class);
+    private final Logger logger = AlertLoggerFactory.getLogger(getClass());
     private final AuthorizationManager authorizationManager;
     private final ConfigContextEnum context;
     private final DescriptorKey descriptorKey;
 
+    private static final String CRUD_ACTION_TEMPLATE = "CRUD action {}: ";
+    private static final String ACTION_CALLED_TEMPLATE = CRUD_ACTION_TEMPLATE + "{} called.";
+    private static final String ACTION_SUCCESS_TEMPLATE = CRUD_ACTION_TEMPLATE + "{} completed successfully.";
+    private static final String ACTION_NOT_FOUND_TEMPLATE = CRUD_ACTION_TEMPLATE + "{} not found.";
+    private static final String ACTION_BAD_REQUEST_TEMPLATE = CRUD_ACTION_TEMPLATE + "{} bad request.";
+    private static final String ACTION_MISSING_PERMISSIONS_TEMPLATE = CRUD_ACTION_TEMPLATE + "{} missing permissions.";
+
     public ConfigurationCrudHelper(AuthorizationManager authorizationManager, ConfigContextEnum context, DescriptorKey descriptorKey) {
         this.authorizationManager = authorizationManager;
         this.context = context;
         this.descriptorKey = descriptorKey;
     }
 
     public <T extends Obfuscated<T>> ActionResponse<T> getOne(Supplier<Optional<T>> modelSupplier) {
+        String actionName = "getOne";
+        logger.trace(ACTION_CALLED_TEMPLATE, descriptorKey.getUniversalKey(), actionName);
```

Comment:
Do we want trace or debug? Since this is the config API, it shouldn't spam the logs too much and will probably be helpful for debugging

Reply:
this is in regards to all of the logs in this class

Reply:
I bumped it down to trace from debug by an earlier request. It does still make a pretty decent number of log messages.  Each page we navigate to using the new controller will create at least 2 messages due to the get request we make.

Reply:
This logger will write to the audit file regardless of log level. So if this is  to be primarily used by us it shouldn't matter what level it's at assuming we're checking that file.

Reply:
I think we want this at trace because it is logging that we entered and exited the method.  I like this in the audit log.  In our UI we have the auto-refresh so if it was at debug we might see a number of get requests because the user is just sitting on a page in the UI with autorefresh.  Might distract from when a user actually clicks on save, test, or delete